docs/docs_maintainer_rules.md

```diff
@@ -22,7 +22,7 @@ Commit types used (sorted by priority):
 * tests - changes in tests.
 * perf - changes in code that target improving performance.
 * docs - changes in text files.
-* refact - changes in code that target improving readability.
+* refact - changes in code that target improving readability or rewriting code in different lang/style.
```

Comment:
Formally smth like rewriting methods to class or vice-versa not covered by this wording, but still understandable.

BTW probably "improving readability" already includes rewriting something from JS to TS